airbyte-cdk/python/airbyte_cdk/sources/declarative/requesters/requester.py

```diff
@@ -87,18 +81,6 @@ def should_retry(self, response: requests.Response) -> bool:
         Unexpected but transient exceptions (connection timeout, DNS resolution failed, etc..) are retried by default.
         """
 
-    @abstractmethod
-    def backoff_time(self, response: requests.Response) -> Optional[float]:
```

Comment:
we don't need this method since `should_retry` will also return the backoff time when needed